lib/worker.js

```diff
@@ -308,14 +309,16 @@ function _gossipWith(ledgerNode, voter, blockHeight, peer, callback) {
       status.consensusPhase)) {
       // TODO: send votes we know about to peer to optimize
 
-      // TODO: validate `status.election`
       // TODO: make sure `Events` election topic is certified *BEFORE*
       //   roll-call because we depend on the votes for events to exist
       //   in the database (and they only get there if they are certified)
-      return async.eachSeries(status.election, (election, callback) => {
-        api._election.certify(
-          ledgerNode, election.topic, election.electionResults, callback);
-      }, callback);
+      return async.series([
+        callback => validate('continuity.election', status.election, callback),
```

Comment:
@dlongley it was suggested that by adding validation to the client get block status API, that validation here may be unnecessary.  Is that true?  Does all the election information pass through that API?  Don't we have an issue for adding an API to send election data via a POST handler?

Reply:
Let's just leave it for now. We can look into optimizing it away later.